lib/origen_doc_helpers/helpers.rb

```diff
@@ -272,7 +272,12 @@ def _resolve_tab(options)
             if options[:tab]
               options[:tab]
             else
-              rel = options[:top_level_file].relative_path_from(_doc_root_dir(options)).sub_ext('').sub_ext('').to_s
+              rel = '..'
```

Comment:
Do you need this line at all if it gets set in `rescue`?

Reply:
Unless I'm getting my programming language rules mixed up, rel has to be created outside of the code block. Otherwise it will go out of scope after the code block. I figured that I might as well set it to '..' when I create it.

Reply:
I guess I could assign rel to the result of the code block instead